trax/data/tf_inputs.py

```diff
@@ -1090,30 +1095,136 @@ def download_and_prepare(dataset_name, data_dir):
   return data_dir
 
 
-def bert_single_sentence_preprocess(batch):
-  for sent1, label in batch:
-    value_vector = np.concatenate(([101], sent1, [102]))
-    segment_embs = np.zeros(sent1.shape[0] + 2, dtype=np.int32)
-    yield value_vector, segment_embs, segment_embs, label, np.int64(1)
+def BertSingleSentenceInputs(batch, labeled=True, CLS_id=101, SEP_id=102):
+  """Prepares inputs for BERT models by adding [SEP] and [CLS] tokens and creating segment embeddings."""
+  if labeled:
+    for sent1, label in batch:
+      value_vector = np.concatenate(([CLS_id], sent1, [SEP_id]))
+      segment_embs = np.zeros(sent1.shape[0] + 2, dtype=np.int32)
+      yield value_vector, segment_embs, segment_embs, label, np.int64(1)
```

Comment:
why `np.int64(1)` and not `np.int32(1)` -- will it work with int32? If so, lets do that.

Reply:
Valid point, I changed it to np.uint32 (these weights are never negative anyway). I also changed segment_embs to np.uint16, because they are tiny (right now never even greater than 1)

Reply:
Let's not do uints -- I'm not sure if we handle that correctly across tf/jax/tf-np etc ... Or what will happen if something downstream assumes a uint32 is an int32 ... let's stick to np.int32 everywhere we can. Thanks!

Reply:
okay, i changed them all to int32 :-)